frontend/src/pages/store/index.svelte

```diff
@@ -426,6 +473,96 @@
 		<h1 class="text-3xl font-medium mb-2">New Arrivals</h1>
 		<NewArrivals />
 	</div>
+
+	<div class="px-12 my-6 flex flex-col w-screen">
+		<h1 class="text-3xl font-medium mb-2">Purchase requests</h1>
+		<div
+			class="flex bg-base-100 shadow-md rounded-lg min-h-40 overflow-x-auto"
+		>
+			{#if !requests}
+				<span
+					class="text-center text-3xl flex justify-center items-center w-full h-full"
+				>
+					Loading...
+				</span>
+			{:else if requests.length > 0}
+				<table class="w-full table-auto">
+					<thead class="w-full">
+						<tr class="text-left border-b border-gray-300">
+							<th class="p-4" />
+							<th class="p-4">Date</th>
+							<th class="p-4">Store</th>
+							<th class="p-4">Item</th>
+							<th class="p-4">Quantity</th>
+							<th class="p-4">Price</th>
+							<th class="p-4" />
+						</tr>
+					</thead>
+					<tbody class="w-full divide-y divide-gray-300">
+						{#if pendingRequests.length > 0}
+							{#each pendingRequests as request}
+								<RequestRow
+									{request}
+									{toastContainer}
+									staff={false}
+									on:reload={loadRequests}
+								/>
+							{/each}
+						{:else}
+							<tr>
+								<td colspan="7" class="py-8">
+									<span
+										class="text-center text-2xl flex justify-center items-center w-full h-full"
+									>
+										No pending requests.
+									</span>
+								</td>
+							</tr>
+						{/if}
+						{#if completedRequests.length > 0}
+							<tr>
+								<td colspan="7" class="p-2">
+									<div class="collapse collapse-arrow">
+										<input
+											type="checkbox"
+											id="requestcollapse"
+											bind:checked={requestCollapse}
+										/>
+										<label
+											for="requestcollapse"
+											class="rounded-lg text-xl font-medium collapse-title !bg-base-200"
+										>
+											{requestCollapse ? 'Hide' : 'Show'} completed
+											requests
+										</label>
+									</div>
+								</td>
+							</tr>
+							<div
+								class="contents"
+								class:visibility-collapse={!requestCollapse}
+							>
+								{#each completedRequests as request}
+									<RequestRow
+										{request}
+										{toastContainer}
+										staff={false}
+										on:reload={loadRequests}
+									/>
+								{/each}
+							</div>
+						{/if}
+					</tbody>
+				</table>
+			{:else}
+				<span
+					class="text-center text-3xl flex justify-center items-center w-full h-full"
```

Comment:
```suggestion
					class="text-center text-3xl flex justify-center items-center w-full h-full py-10"
```
The text shown before a user has made any requests doesn't match the new arrivals section's sizing which looks odd
![Screenshot from 2022-05-07 12-30-58](https://user-images.githubusercontent.com/37093293/167266536-99c678fb-7b1c-43b6-ae09-3a29fc6728f1.png)